common/containers/Tabs/Swap/components/LiteSend/Fields.tsx

```diff
@@ -103,6 +103,9 @@ class FieldsClass extends Component<Props> {
 }
 
 export const Fields = connect(
-  (state: AppState) => ({ unit: getUnit(state), currentBalance: getCurrentBalance(state) }),
+  (state: AppState) => ({
```

Comment:
Prettier conflict?

Reply:
Not sure, I might have just indented for some reason while I was tinkering in here but ultimately made no changes.